.vitepress/config.mts

```diff
@@ -30,94 +30,89 @@ export default defineConfig({
         {
           link: "roles",
           text: "Roles and Permissions",
+          items: [],
         },
```

Comment:
Maybe nest this section somewhere else?

Or leave it as:

```suggestion
        {
          text: "Roles and Permissions",
          items: [{ text: 'Overview', link: '/roles' }],
        },
```

Having a sole clickable top-level sidebar item is not consistent with the other sections.